Do not create new release in release branch automatically

Felixoid opened the pull request:
<!---
A technical comment, you are free to remove or leave it as it is when PR is created
The following categories are used in the next scripts, update them accordingly
utils/changelog/changelog.py
tests/ci/cancel_and_rerun_workflow_lambda/app.py
-->
### Changelog category (leave one):
- CI Fix or Improvement (changelog entry is not required)


### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add `ClickHouseVersion.copy` method. Create a branch release in advance without spinning out the release to increase the stability.

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

<!---
Directly edit documentation source files in the "docs" folder with the same pull-request as code changes

or

Add a user-readable short description of the changes that should be added to docs.clickhouse.com below.

At a minimum, the following information should be added (but add more as needed).
- Motivation: Why is this function, table engine, etc. useful to ClickHouse users?

- Parameters: If the feature being added takes arguments, options or is influenced by settings, please list them below with a brief explanation.

- Example use: A query or command.
-->


> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

<details>
    <summary>Modify your CI run</summary>

**NOTE:** If your merge the PR with modified CI you **MUST KNOW** what you are doing
**NOTE:** Checked options will be applied if set before CI RunConfig/PrepareRunConfig step

#### Include tests (required builds will be added automatically):
- [ ] <!---ci_include_fast--> Fast test
- [ ] <!---ci_include_integration--> Integration Tests
- [ ] <!---ci_include_stateless--> Stateless tests
- [ ] <!---ci_include_stateful--> Stateful tests
- [ ] <!---ci_include_unit--> Unit tests
- [ ] <!---ci_include_performance--> Performance tests
- [ ] <!---ci_include_asan--> All with ASAN
- [ ] <!---ci_include_tsan--> All with TSAN
- [ ] <!---ci_include_analyzer--> All with Analyzer
- [ ] <!---ci_include_azure --> All with Azure
- [ ] <!---ci_include_KEYWORD--> Add your option here

#### Exclude tests:
- [ ] <!---ci_exclude_fast--> Fast test
- [ ] <!---ci_exclude_integration--> Integration Tests
- [ ] <!---ci_exclude_stateless--> Stateless tests
- [ ] <!---ci_exclude_stateful--> Stateful tests
- [ ] <!---ci_exclude_performance--> Performance tests
- [ ] <!---ci_exclude_asan--> All with ASAN
- [ ] <!---ci_exclude_tsan--> All with TSAN
- [ ] <!---ci_exclude_msan--> All with MSAN
- [ ] <!---ci_exclude_ubsan--> All with UBSAN
- [ ] <!---ci_exclude_coverage--> All with Coverage
- [ ] <!---ci_exclude_aarch64--> All with Aarch64
- [ ] <!---ci_exclude_KEYWORD--> Add your option here

#### Extra options:
- [ ] <!---do_not_test--> do not test (only style check)
- [ ] <!---no_merge_commit--> disable merge-commit (no merge from master before tests)
- [ ] <!---no_ci_cache--> disable CI cache (job reuse)

#### Only specified batches in multi-batch jobs:
- [ ] <!---batch_0--> 1
- [ ] <!---batch_1--> 2
- [ ] <!---batch_2--> 3
- [ ] <!---batch_3--> 4

</details>

Felixoid committed:
Do not create new release in release branch automatically

robot-ch-test-poll1 commented:
<!-- automatic status comment for PR #64039 from ClickHouse/ClickHouse:new-release-create-branch-only -->
*This is an automated comment for commit 803a52ba944fb00b8a3a266317f50de063dd534f with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/64039/803a52ba944fb00b8a3a266317f50de063dd534f/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

Felixoid commented:
<details><summary>Here's the run example</summary>

```
>±> ./release.py --dry-run --with-stderr --commit 2e76275124d851ee91fff7d7ead5cef410dd1148 --type new --no-run-from-master
INFO:root:Running command:
    gh auth status
INFO:root:Running command:
    gh api 'repos/ClickHouse/ClickHouse/commits/2e76275124d851ee91fff7d7ead5cef410dd1148/statuses?per_page=100&page=1'                                                                                                                                                                                                           INFO:root:Checking if repo is clean
INFO:root:Running command:
    git diff HEAD --exit-code
INFO:root:Running command:
    git fetch git@github.com:ClickHouse/ClickHouse.git 2e76275124d851ee91fff7d7ead5cef410dd1148 --no-recurse-submodules
From github.com:ClickHouse/ClickHouse
 * branch                    2e76275124d851ee91fff7d7ead5cef410dd1148 -> FETCH_HEAD
INFO:root:Running command:
    git fetch git@github.com:ClickHouse/ClickHouse.git --tags --no-recurse-submodules
From github.com:ClickHouse/ClickHouse
 * branch                    HEAD       -> FETCH_HEAD
INFO:root:Running command:
    git checkout 2e76275124d851ee91fff7d7ead5cef410dd1148
Note: switching to '2e76275124d851ee91fff7d7ead5cef410dd1148'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 2e76275124d Merge pull request #63202 from hanfei1991/hanfei/fix-intersection
INFO:root:Running command:
    git checkout new-release-create-branch-only
Previous HEAD position was 2e76275124d Merge pull request #63202 from hanfei1991/hanfei/fix-intersection
Switched to branch 'new-release-create-branch-only'
INFO:root:Running command:
    git fetch git@github.com:ClickHouse/ClickHouse.git master:master --no-recurse-submodules
INFO:root:Running command:
    git branch --contains=2e76275124d851ee91fff7d7ead5cef410dd1148 master
INFO:root:Running command:
    git checkout 2e76275124d851ee91fff7d7ead5cef410dd1148
Note: switching to '2e76275124d851ee91fff7d7ead5cef410dd1148'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 2e76275124d Merge pull request #63202 from hanfei1991/hanfei/fix-intersection
INFO:root:Running command:
    git branch 24.6-prepare 2e76275124d851ee91fff7d7ead5cef410dd1148
INFO:root:Running command:
    git checkout 24.6-prepare
Switched to branch '24.6-prepare'
INFO:root:Running command:
    git diff '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/cmake/autogenerated_versions.txt' '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/src/Storages/System/StorageSystemContributors.generated.cpp'
INFO:root:Dry running, resetting the following changes in the repo:
diff --git a/cmake/autogenerated_versions.txt b/cmake/autogenerated_versions.txt
index f8ff71876c6..4443adf8058 100644
--- a/cmake/autogenerated_versions.txt
+++ b/cmake/autogenerated_versions.txt
@@ -2,11 +2,11 @@

 # NOTE: has nothing common with DBMS_TCP_PROTOCOL_VERSION,
 # only DBMS_TCP_PROTOCOL_VERSION should be incremented on protocol changes.
-SET(VERSION_REVISION 54486)
+SET(VERSION_REVISION 54487)
 SET(VERSION_MAJOR 24)
-SET(VERSION_MINOR 5)
+SET(VERSION_MINOR 6)
 SET(VERSION_PATCH 1)
-SET(VERSION_GITHASH 6d4b31322d168356c8b10c43b4cef157c82337ff)
-SET(VERSION_DESCRIBE v24.5.1.1-testing)
-SET(VERSION_STRING 24.5.1.1)
+SET(VERSION_GITHASH 2e76275124d851ee91fff7d7ead5cef410dd1148)
+SET(VERSION_DESCRIBE v24.6.1.1-testing)
+SET(VERSION_STRING 24.6.1.1)
 # end of autochange
diff --git a/src/Storages/System/StorageSystemContributors.generated.cpp b/src/Storages/System/StorageSystemContributors.generated.cpp
index 909599c00af..ba98c653094 100644
--- a/src/Storages/System/StorageSystemContributors.generated.cpp
+++ b/src/Storages/System/StorageSystemContributors.generated.cpp
@@ -48,6 +48,7 @@ const char * auto_contributors[] {
     "Alex Cao",
     "Alex Cheng",
     "Alex Karo",
+    "Alex Katsman",
     "Alex Krash",
     "Alex Ryndin",
     "Alex Zatelepin",
@@ -101,6 +102,7 @@ const char * auto_contributors[] {
     "Alexey Korepanov",
     "Alexey Milovidov",
     "Alexey Perevyshin",
+    "Alexey Petrunyaka",
     "Alexey Tronov",
     "Alexey Vasiliev",
     "Alexey Zatelepin",
@@ -109,6 +111,7 @@ const char * auto_contributors[] {
     "AlfVII",
     "Alfonso Martinez",
     "Alfred Xu",
+    "Ali",
     "Ali Demirci",
     "Aliaksandr Pliutau",
     "Aliaksandr Shylau",
@@ -384,6 +387,7 @@ const char * auto_contributors[] {
     "Evgenii Pravda",
     "Evgeniia Sudarikova",
     "Evgeniy Gatov",
+    "Evgeniy Leko",
     "Evgeniy Udodov",
     "Evgeny",
     "Evgeny Konkov",
@@ -446,6 +450,7 @@ const char * auto_contributors[] {
     "Grigory Buteyko",
     "Grigory Pervakov",
     "GruffGemini",
+    "Grégoire Pineau",
     "Guillaume Tassery",
     "Guo Wangyang",
     "Guo Wei (William)",
@@ -640,6 +645,7 @@ const char * auto_contributors[] {
     "Leonardo Maciel",
     "Leonid Krylov",
     "Leopold Schabel",
+    "Leticia Webb",
     "Lev Borodin",
     "Lewinma",
     "Li Shuai",
@@ -701,6 +707,7 @@ const char * auto_contributors[] {
     "Masha",
     "Mathieu Rey",
     "Matthew Peveler",
+    "Mattias Naarttijärvi",
     "Matwey V. Kornilov",
     "Max",
     "Max Akhmedov",
@@ -711,6 +718,7 @@ const char * auto_contributors[] {
     "MaxTheHuman",
     "MaxWk",
     "Maxim Akhmedov",
+    "Maxim Alexeev",
     "Maxim Babenko",
     "Maxim Fedotov",
     "Maxim Fridental",
@@ -739,6 +747,7 @@ const char * auto_contributors[] {
     "Michael Razuvaev",
     "Michael Schnerring",
     "Michael Smitasin",
+    "Michael Stetsyuk",
     "Michail Safronov",
     "Michal Lisowski",
     "MicrochipQ",
@@ -1123,6 +1132,7 @@ const char * auto_contributors[] {
     "Victor Krasnov",
     "Victor Tarnavsky",
     "Viktor Taranenko",
+    "Vinay Suryadevara",
     "Vincent",
     "Vincent Bernat",
     "Vitalii S",
@@ -1162,6 +1172,9 @@ const char * auto_contributors[] {
     "Vladislav Smirnov",
     "Vladislav V",
     "Vojtech Splichal",
+    "Volodya",
+    "Volodya Giro",
+    "Volodyachan",
     "Volodymyr Kuznetsov",
     "Vsevolod Orlov",
     "Vxider",
@@ -1179,6 +1192,7 @@ const char * auto_contributors[] {
     "XenoAmess",
     "Xianda Ke",
     "Xiang Zhou",
+    "Xiaofei Hu",
     "Xin Wang",
     "Xoel Lopez Barata",
     "Xudong Zhang",
@@ -1490,6 +1504,7 @@ const char * auto_contributors[] {
     "jiyoungyoooo",
     "jktng",
     "jkuklis",
+    "joe09@foxmail.com",
     "joelynch",
     "johanngan",
     "johnnymatthews",
@@ -1658,6 +1673,7 @@ const char * auto_contributors[] {
     "ongkong",
     "orantius",
     "p0ny",
+    "p1rattttt",
     "palasonicq",
     "palegre-tiny",
     "pawelsz-rb",
@@ -1667,6 +1683,7 @@ const char * auto_contributors[] {
     "pedro.riera",
     "pengxiangcai",
     "peshkurov",
+    "pet74alex",
     "peter279k",
     "philip.han",
     "pingyu",
@@ -1802,6 +1819,7 @@ const char * auto_contributors[] {
     "vic",
     "vicdashkov",
     "vicgao",
+    "vinay92-ch",
     "vinity",
     "vitac",
     "vitstn",
@@ -1877,6 +1895,7 @@ const char * auto_contributors[] {
     "zhenjial",
     "zhifeng",
     "zhongyuankai",
+    "zhou",
     "zhoubintao",
     "zhukai",
     "zimv",
INFO:root:Running command:
    git checkout '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/cmake/autogenerated_versions.txt' '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/src/Storages/System/StorageSystemContributors.generated.cpp'
Updated 2 paths from the index
INFO:root:Would run command:
    git commit '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/cmake/autogenerated_versions.txt' '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/src/Storages/System/StorageSystemContributors.generated.cpp' -m 'Update autogenerated version to 24.6.1.1 and contributors'
INFO:root:Would run command:
    git push git@github.com:ClickHouse/ClickHouse.git 24.6-prepare:24.6-prepare
INFO:root:Would run command:
    gh pr create --repo ClickHouse/ClickHouse --title 'Update version after release' --head 24.6-prepare --body-file '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/.github/PULL_REQUEST_TEMPLATE.md' --label 'do not test' --assignee @me
INFO:root:Running command:
    git tag --contains=2e76275124d851ee91fff7d7ead5cef410dd1148
INFO:root:Running command:
    git branch 24.5 2e76275124d851ee91fff7d7ead5cef410dd1148
INFO:root:Running command:
    git checkout 24.5
Switched to branch '24.5'
INFO:root:Running command:
    git diff '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/cmake/autogenerated_versions.txt' '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/src/Storages/System/StorageSystemContributors.generated.cpp'
INFO:root:Dry running, resetting the following changes in the repo:
diff --git a/cmake/autogenerated_versions.txt b/cmake/autogenerated_versions.txt
index f8ff71876c6..74c1d6f0968 100644
--- a/cmake/autogenerated_versions.txt
+++ b/cmake/autogenerated_versions.txt
@@ -6,7 +6,7 @@ SET(VERSION_REVISION 54486)
 SET(VERSION_MAJOR 24)
 SET(VERSION_MINOR 5)
 SET(VERSION_PATCH 1)
-SET(VERSION_GITHASH 6d4b31322d168356c8b10c43b4cef157c82337ff)
-SET(VERSION_DESCRIBE v24.5.1.1-testing)
+SET(VERSION_GITHASH 2e76275124d851ee91fff7d7ead5cef410dd1148)
+SET(VERSION_DESCRIBE v24.5.1.1-stable)
 SET(VERSION_STRING 24.5.1.1)
 # end of autochange
diff --git a/src/Storages/System/StorageSystemContributors.generated.cpp b/src/Storages/System/StorageSystemContributors.generated.cpp
index 909599c00af..ba98c653094 100644
--- a/src/Storages/System/StorageSystemContributors.generated.cpp
+++ b/src/Storages/System/StorageSystemContributors.generated.cpp
@@ -48,6 +48,7 @@ const char * auto_contributors[] {
     "Alex Cao",
     "Alex Cheng",
     "Alex Karo",
+    "Alex Katsman",
     "Alex Krash",
     "Alex Ryndin",
     "Alex Zatelepin",
@@ -101,6 +102,7 @@ const char * auto_contributors[] {
     "Alexey Korepanov",
     "Alexey Milovidov",
     "Alexey Perevyshin",
+    "Alexey Petrunyaka",
     "Alexey Tronov",
     "Alexey Vasiliev",
     "Alexey Zatelepin",
@@ -109,6 +111,7 @@ const char * auto_contributors[] {
     "AlfVII",
     "Alfonso Martinez",
     "Alfred Xu",
+    "Ali",
     "Ali Demirci",
     "Aliaksandr Pliutau",
     "Aliaksandr Shylau",
@@ -384,6 +387,7 @@ const char * auto_contributors[] {
     "Evgenii Pravda",
     "Evgeniia Sudarikova",
     "Evgeniy Gatov",
+    "Evgeniy Leko",
     "Evgeniy Udodov",
     "Evgeny",
     "Evgeny Konkov",
@@ -446,6 +450,7 @@ const char * auto_contributors[] {
     "Grigory Buteyko",
     "Grigory Pervakov",
     "GruffGemini",
+    "Grégoire Pineau",
     "Guillaume Tassery",
     "Guo Wangyang",
     "Guo Wei (William)",
@@ -640,6 +645,7 @@ const char * auto_contributors[] {
     "Leonardo Maciel",
     "Leonid Krylov",
     "Leopold Schabel",
+    "Leticia Webb",
     "Lev Borodin",
     "Lewinma",
     "Li Shuai",
@@ -701,6 +707,7 @@ const char * auto_contributors[] {
     "Masha",
     "Mathieu Rey",
     "Matthew Peveler",
+    "Mattias Naarttijärvi",
     "Matwey V. Kornilov",
     "Max",
     "Max Akhmedov",
@@ -711,6 +718,7 @@ const char * auto_contributors[] {
     "MaxTheHuman",
     "MaxWk",
     "Maxim Akhmedov",
+    "Maxim Alexeev",
     "Maxim Babenko",
     "Maxim Fedotov",
     "Maxim Fridental",
@@ -739,6 +747,7 @@ const char * auto_contributors[] {
     "Michael Razuvaev",
     "Michael Schnerring",
     "Michael Smitasin",
+    "Michael Stetsyuk",
     "Michail Safronov",
     "Michal Lisowski",
     "MicrochipQ",
@@ -1123,6 +1132,7 @@ const char * auto_contributors[] {
     "Victor Krasnov",
     "Victor Tarnavsky",
     "Viktor Taranenko",
+    "Vinay Suryadevara",
     "Vincent",
     "Vincent Bernat",
     "Vitalii S",
@@ -1162,6 +1172,9 @@ const char * auto_contributors[] {
     "Vladislav Smirnov",
     "Vladislav V",
     "Vojtech Splichal",
+    "Volodya",
+    "Volodya Giro",
+    "Volodyachan",
     "Volodymyr Kuznetsov",
     "Vsevolod Orlov",
     "Vxider",
@@ -1179,6 +1192,7 @@ const char * auto_contributors[] {
     "XenoAmess",
     "Xianda Ke",
     "Xiang Zhou",
+    "Xiaofei Hu",
     "Xin Wang",
     "Xoel Lopez Barata",
     "Xudong Zhang",
@@ -1490,6 +1504,7 @@ const char * auto_contributors[] {
     "jiyoungyoooo",
     "jktng",
     "jkuklis",
+    "joe09@foxmail.com",
     "joelynch",
     "johanngan",
     "johnnymatthews",
@@ -1658,6 +1673,7 @@ const char * auto_contributors[] {
     "ongkong",
     "orantius",
     "p0ny",
+    "p1rattttt",
     "palasonicq",
     "palegre-tiny",
     "pawelsz-rb",
@@ -1667,6 +1683,7 @@ const char * auto_contributors[] {
     "pedro.riera",
     "pengxiangcai",
     "peshkurov",
+    "pet74alex",
     "peter279k",
     "philip.han",
     "pingyu",
@@ -1802,6 +1819,7 @@ const char * auto_contributors[] {
     "vic",
     "vicdashkov",
     "vicgao",
+    "vinay92-ch",
     "vinity",
     "vitac",
     "vitstn",
@@ -1877,6 +1895,7 @@ const char * auto_contributors[] {
     "zhenjial",
     "zhifeng",
     "zhongyuankai",
+    "zhou",
     "zhoubintao",
     "zhukai",
     "zimv",
INFO:root:Running command:
    git checkout '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/cmake/autogenerated_versions.txt' '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/src/Storages/System/StorageSystemContributors.generated.cpp'
Updated 2 paths from the index
INFO:root:Would run command:
    git commit '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/cmake/autogenerated_versions.txt' '/home/felixoid/Space/Felixoid/github/ClickHouse/ClickHouse/src/Storages/System/StorageSystemContributors.generated.cpp' -m 'Update autogenerated version to 24.5.1.1 and contributors'
INFO:root:Would run command:
    git push git@github.com:ClickHouse/ClickHouse.git 24.5:24.5
INFO:root:Would run command:
    gh api repos/ClickHouse/ClickHouse/labels -f name=v24.5-must-backport -f color=10dbed
INFO:root:Would run command:
    gh api repos/ClickHouse/ClickHouse/labels -f name=v24.5-affected -f color=c2bfff
INFO:root:Would run command:
    gh pr create --repo ClickHouse/ClickHouse --title 'Release pull request for branch 24.5' --head 24.5 --label release --body 'This PullRequest is a part of ClickHouse release cycle. It is used by CI system only. Do not perform any changes with it.'
INFO:root:Publishing release v24.5.1.1216-stable from commit 2e76275124d851ee91fff7d7ead5cef410dd1148 is done
INFO:root:Running command:
    git checkout 24.6-prepare
Switched to branch '24.6-prepare'
INFO:root:Running command:
    git checkout 2e76275124d851ee91fff7d7ead5cef410dd1148
Note: switching to '2e76275124d851ee91fff7d7ead5cef410dd1148'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 2e76275124d Merge pull request #63202 from hanfei1991/hanfei/fix-intersection
INFO:root:Running command:
    git checkout new-release-create-branch-only
Previous HEAD position was 2e76275124d Merge pull request #63202 from hanfei1991/hanfei/fix-intersection
Switched to branch 'new-release-create-branch-only'
INFO:root:Dry running, clean out possible changes
INFO:root:Running command:
    # gh api repos/ClickHouse/ClickHouse/labels/v24.5-affected -X DELETE
INFO:root:Running command:
    # gh api repos/ClickHouse/ClickHouse/labels/v24.5-must-backport -X DELETE
INFO:root:Running command:
    # git push -d git@github.com:ClickHouse/ClickHouse.git 24.5
INFO:root:Running command:
    git branch -D 24.5
INFO:root:Running command:
    # git push -d git@github.com:ClickHouse/ClickHouse.git 24.6-prepare
INFO:root:Running command:
    git branch -D 24.6-prepare
```

</details>

Felixoid commented:
<details><summary>The patch releases works too</summary>

```
m/StorageSystemContributors.generated.cpp' -m 'Update autogenerated version to 24.3.4.1 and contributors'
INFO:root:Would run command:
    git push git@github.com:ClickHouse/ClickHouse.git HEAD:24.3
INFO:root:Publishing release v24.3.4.63-lts from commit dcb26df0f54d86e976d8930c384ea4f4c8333817 is done
INFO:root:Running command:
    git checkout new-release-create-branch-only
warning: unable to rmdir 'contrib/boringssl': Directory not empty
Previous HEAD position was dcb26df0f54 Merge pull request #64009 from ClickHouse/backport/24.3/63930
Switched to branch 'new-release-create-branch-only'
INFO:root:Dry running, clean out possible changes
INFO:root:Running command:
    # git push git@github.com:ClickHouse/ClickHouse.git +dcb26df0f54d86e976d8930c384ea4f4c8333817:24.3
INFO:root:Running command:
    # gh release delete --yes --repo ClickHouse/ClickHouse 'v24.3.4.63-lts'
INFO:root:Running command:
    # git push -d git@github.com:ClickHouse/ClickHouse.git v24.3.4.63-lts
INFO:root:Running command:
    # git tag -d 'v24.3.4.63-lts'
```

</details>
